Application/Telegram/MainActions/NewBookingCommand.cs

```diff
@@ -92,7 +92,7 @@ public static async Task HandleAsync(Update update, TelegramBotClient telegraBot
                     DateHelper.EndMonthUpdater(update.CallbackQuery, ref user);
                     await new SendDayCommand(mediator, telegraBotClient).SendAsync(
                         update.CallbackQuery
-                        ,"Please Select Workplace Booking Type "
+                        , "Please select end date day"
```

Comment:
why are you touching 'NewBookingCommand' ? you need to implement it in 'NewVacationCommand'

Reply:
We are sending day and our bot should send message like select end date day not select booking type right ?

Reply:
yes, you are right